src/app.js

```diff
@@ -80,9 +82,10 @@ function startApp(d2) {
 // can use it to access the api, translations etc.
 getManifest('./manifest.webapp')
     .then(manifest => {
-        const baseUrl = process.env.NODE_ENV === 'production' ? manifest.getBaseUrl() : dhisDevConfig.baseUrl;
+        const baseUrl = process.env.NODE_ENV === 'production' ?
+            manifest.getBaseUrl() : dhisDevConfig.baseUrl;
+        config.siteUrl = baseUrl;
```

Comment:
Internally, right. Note that this `config` comes from `import { init, config, getUserSettings, getManifest } from 'd2/lib/d2';`, it's a internal structure of d2.